airbyte-integrations/connectors/source-slack/source_slack/source.py

```diff
@@ -285,16 +284,36 @@ def request_body_json(self, stream_slice: Mapping = None, **kwargs) -> Optional[
 
 
 class SourceSlack(AbstractSource):
+    def _get_authenticator(self, config: Mapping[str, Any]):
+        credentials = config.get("credentials")
+        credentials_title = credentials.get("option_title")
+        if credentials_title == "Default OAuth2.0 authorization":
```

Comment:
Aren't option_title omiited by webui/airbyte worker when generating config file? IMO its more reliable to check if "client_id" key present in credentials and remove  option_title from required parameters

Reply:
Not really, this parameter is declared as required, and is set as a constant, so that in the case of `oneOf`, we can understand what configuration has come to us, and depending on this, take further actions.

Reply:
Are you sure webui behaves like that? It not a straightforward. Have you tested it on real app?

Reply:
Yep, worked with this before